syntax = "proto3";

package limestone.grpc.testing;

service PingService {
  rpc Ping (PingRequest) returns (PingResponse);
}

message PingRequest {}
message PingResponse {}
